clusterloader2/pkg/util/util.go

```diff
@@ -217,3 +217,14 @@ func CopyMap(src, dest map[string]interface{}) {
 		dest[k] = v
 	}
 }
+
+// IsChannelClosed returns true if given channel is closed.
+func IsChannelClosed(ch <-chan struct{}) bool {
+	select {
+	case <-ch:
```

Comment:
This isn't correct - if the channel is open but something is writing to it (or something is already buffered) it will also return true. But channel is not closed.
It's not really possible to check it efficiently.

Let's just remove it and assume channel is not closed - this class is not thread-safe anyway.

Reply:
Removed.